packages/webapp/src/pages/induction/[id].tsx

```diff
@@ -1,128 +1,106 @@
+    } = useGetInductionWithEndorsements(inductionId as string);
 
-    const endorsements = inductionEndorsements
-        ? inductionEndorsements.endorsements
-        : [];
+    const isLoading = isLoadingEndorsements || isLoadingCommunityState;
 
-    // TODO: Consider deriving the user's role here and return an enum of roles: INVITER, ENDORSER, INVITEE, MEMBER, EOS_USER, UNAUTHENTICATED.
-    // Almost every child component of this page cares about the role of the user in relation to the invite/induction. We can pass role down.
-    const isEndorser = useMemo(() => 
-         endorsements.some(
-            (endorsement) => endorsement.endorser === ualAccount?.accountName
-        )
-    , [ualAccount, endorsements]);
+    const induction = data?.induction;
+    const endorsements = data?.endorsements ?? [];
 
-    const status = getInductionStatus(induction);
+    const userRole = useInductionUserRole(endorsements, induction);
+    const status = getInductionStatus(induction, endorsements);
 
-    const renderInductionStep = useMemo(() => {
+    const renderInductionJourney = useMemo(() => {
         if (!induction) return "";
-
-        if (reviewStep === "profile") {
-            return (
-                <InductionStepProfile
-                    induction={induction}
-                    isCommunityActive={isCommunityActive}
-                    isEndorser={isEndorser}
-                    isReviewing
-                />
-            );
-        }
-
-        if (reviewStep === "video") {
-            return (
-                <InductionStepVideo
-                    induction={induction}
-                    isEndorser={isEndorser}
-                    isReviewing
-                />
-            );
-        }
-
-        switch (status) {
-            case InductionStatus.waitingForProfile:
+        switch (userRole) {
+            case InductionRole.Inviter:
+            case InductionRole.Endorser:
                 return (
-                    <InductionStepProfile
+                    <InviterWitnessJourney
+                        endorsements={endorsements}
                         induction={induction}
-                        isEndorser={isEndorser}
-                        isCommunityActive={isCommunityActive}
+                        inductionStatus={status}
                     />
                 );
-            case InductionStatus.waitingForVideo:
+            case InductionRole.Invitee:
                 return (
-                    <InductionStepVideo
+                    <InviteeJourney
+                        endorsements={endorsements}
                         induction={induction}
-                        isEndorser={isEndorser}
+                        inductionStatus={status}
                     />
                 );
-            case InductionStatus.waitingForEndorsement:
+            case InductionRole.Member:
+            case InductionRole.Unknown:
+            case InductionRole.Unauthenticated:
```

Comment:
replace with default... otherwise we need to maintain two places whenever we add a new Role to make sure it can view the ThirdPartyJourney